### PR TITLE
Feature/worker deployment

### LINKED ISSUE #74: Fetch worker deploy scripts
The worker deploy scripts are in a separate [repository](https://github.com/wesleymonte/worker-deployment). It is necessary to move them here.

### PR DESCRIPTION
**Pull request by wesleymonte**
Resolves #74.
I fetch the worker deployment scripts to this repository.

### CODE REVIEW

File: worker/deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# Required (if not specified, ansible will use the host ssh keys)
+ansible_ssh_private_key_file=/home/admin/.ssh/priv_key
+
+# Default
+ansible_files_path=./ansible-playbook
+```
+
+## Install
+
+After configuring the `hosts.conf` file, execute the `install.sh` script to setup the Worker Node.
```
Comment (raonismaneoto) [inline worker/deploy/README.md]: execute it where? in the coordinator host or in the worker node itself?

File: worker/deploy/setup.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# A script to setup a worker environment.
+# It expects to be run on Ubuntu 16.04 via 'sudo'
```
Comment (raonismaneoto) [inline worker/deploy/setup.sh]: It must be runned*

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: worker/deploy/README.md (hunk 1 above)
Comment (wesleymonte) [reply worker/deploy/README.md]: In the coordinator host. I added info about it.

File: worker/deploy/setup.sh (hunk 2 above)
Comment (wesleymonte) [reply worker/deploy/setup.sh]: Done.

**Review by raonismaneoto — APPROVED** [review]

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**